### ISSUE TITLE
Publish on F-Droid

### ISSUE THREAD

**Issue by bandali** [gabrielelucci/greyscale-cmte#6]
Hi there,

Firstly, thanks for the beautiful theme, and thanks for publishing the source code (aside: it's not "open source" yet technically, since you haven't added a LICENSE file).

Some people (myself included) have chosen not to install Google Play services on their devices. It'd be nice to have this app published on [F-Droid](https://f-droid.org/) to make it more accessible.

**Comment by gabrielelucci** [comment]
Hi,

Thank you! I recently changed license from Apache to CC-BY and forgot to re
add the license file, I'll do.

Also, I've been trying for a long time to upload the apk to F-Droid, but I
can't access my account for unknown reasons. Recovering password doesn't
even work, I think I will write soon to F-Droid and see if they can
"unblock" my account.

Gabriele.

Il giorno gio 25 feb 2016, 16:05 Amin Bandali notifications@github.com ha
scritto:

**Comment by bandali** [comment]
Hi!

Thanks for the super quick reply!

> Also, I've been trying for a long time to upload the apk to F-Droid, but I can't access my account for unknown reasons. Recovering password doesn't even work, I think I will write soon to F-Droid and see if they can "unblock" my account.

Oh that's unfortunate :( In the mean time, if I manage to find some free time I'll try to directly open a merge request on their repo to get this theme included.

**Comment by gabrielelucci** [comment]
Actually I found out that you don't need the F-Droid account to do a merge request. So I'll try to do it later.

**Comment by gabrielelucci** [comment]
I'm trying to setup fdroid, but running `fdroid readmeta` results in this error `CRITICAL: Unrecognised field type for Author Name in metadata/xxx.app.txt`.

Can you help me?

**Comment by bandali** [comment]
Hmm, I'm not sure what could be wrong, I haven't seen many apps with the "Author Name" field present.

Can you copy your metadata file here?

Off top of my head I know [Open Flood](https://gitlab.com/fdroid/fdroiddata/blob/master/metadata/com.gunshippenguin.openflood.txt) and [PinPoi](https://gitlab.com/fdroid/fdroiddata/blob/master/metadata/io.github.fvasco.pinpoi.txt) have author names. Maybe looking at their metadata could give you some clues?

**Comment by gabrielelucci** [comment]
Indeed you're right, the actual  app that is causing the error is Open Flood.

`CRITICAL: Unrecognised field type for Author Name in metadata/com.gunshippenguin.openflood.txt:3`

 I previously cut off the name to be more generic. Anyway the metadata file is [here](https://gitlab.com/fdroid/fdroiddata/blob/master/metadata/com.gunshippenguin.openflood.txt). Seems ok to me, but I'm not used to fdroid yet. Also, deleting the file and rerunning the command just points the error to another file the uses "Author Name" metadata.

**Comment by gabrielelucci** [comment]
Okay, I got past this issue by pulling fdroidserver from git instead of installing it from the package manager of my distro. Probably it was an outdated version. Now I'm building the metadata for my own app and let's see what happens!

**Comment by gabrielelucci** [comment]
Done! Sent a merge request to fdroiddata. 

Merge request [#1243](https://gitlab.com/fdroid/fdroiddata/merge_requests/1243)

**Comment by gabrielelucci** [comment]
Actually I realized it fails to build due to wrong gradle version onn build server. I need to find a workaround to this.

**Comment by bandali** [comment]
Awesome, look forward to using it!

Cheers :beers:

**Comment by bandali** [comment]
@gabrielelucci Greyscale landed on F-Droid today, congrats!

P.S. it's gorgeous!

**Comment by gabrielelucci** [comment]
Thank you! I definitely learned something with this, I will surely distribute all my free projects in fdroid! Thank you!

**Comment by bandali** [comment]
You're very welcome, and I'm really glad to hear that.

I look forward to your future projects, as well as updates to this one!
